cmd/crc/cmd/status.go

```diff
@@ -20,7 +20,7 @@ func init() {
 var statusCmd = &cobra.Command{
 	Use:   "status",
 	Short: "Display status of the cluster",
-	Long:  "Show details about OpenShift cluster and crc vm",
+	Long:  "Show details about OpenShift cluster and the CRC VM",
```

Comment:
`Show details about the OpenShift cluster and the CRC VM`

Reply:
VM or instance? CRC or CodeReady Containers ?

Reply:
@cfergeau what would you suggest it to be in full ?